ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -63,8 +63,8 @@
 
 
 # The naming convention is T for Entity Types and C for Create Types
-T = TypeVar("T", bound=BaseModel)
-C = TypeVar("C", bound=BaseModel)
+T = TypeVar("T", bound=BaseModel)  # pylint: disable=invalid-name
+C = TypeVar("C", bound=BaseModel)  # pylint: disable=invalid-name
```

Comment:
These rules should also be in `pylintrc`: good-names=T,C. IDK why that didnt get picked up 🤔